converters/fromZigbee.js

```diff
@@ -8575,6 +8575,89 @@ const converters = {
             return result;
         },
     },
+    centralite_3156105_thermostat: {
```

Comment:
Can you move this converter into `centralite.js` under `fzLocal` and do the same for the changes to `constants.js`? Example: https://github.com/Koenkk/zigbee2mqtt.io/blob/402126e4bac0aa2f84d716bdec4420b3ab55a661/docs/externalConvertersExample/freepad_ext.js#L20

To prevent code duplication, you can remove everything that is the same as in `fz.thermostat` by calling the converter here, example: https://github.com/Koenkk/zigbee-herdsman-converters/blob/c1304800b51c7b90417c8c0be985c66a96d6681f/converters/fromZigbee.js#L6462